README.md

```diff
@@ -55,6 +55,8 @@ DVSA sends receipts in the email (which will help you in hacking it). You can us
 
 **_Note_**: each user will be assigned an email from `mailsac.com` which will be automatically verified. Real emails will be sent to their account and will appear in the applicaiton Inbox page. All this is **transparent** to the user and the deployer).
 
+**_Note_**: to make the email verification script work your dafault AWS region has to be "US East (N. Virginia)", for example by setting `region = us-east-1` in your ~/.aws/config file 
```

Comment:
There's a typo: "dafault" should be "default".

Reply:
Does the script only work with `us-east-1`, or would it work if you used another region?